Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/FilterAdapter.cpp

```diff
@@ -598,15 +598,15 @@ namespace AZ::DocumentPropertyEditor
     {
     }
 
-    void ValueStringFilter::SetFilterString(QString filterString)
+    void ValueStringFilter::SetFilterString(const AZStd::string& filterString)
```

Comment:
This string is being moved on line 605, is it intentional? Or should it be passed by value?

Reply:
good catch! My brain just reverted to const& by default. Fixed now, thanks :)